### PR TITLE
Added info about enabling OES repos.

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Short explanation about enabling OES repositories.
Added to Troubleshooting Guide, SUMA only.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3 https://github.com/uyuni-project/uyuni-docs/pull/1867
- [ ] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/19327.

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Added info about OES repos enablement to Troubleshooting section in 
+  AdminGuide (bsc#1204195)
```
Comment (keichwa) [inline .changelog:8]:
```suggestion
- Added information  about OES repository enablement to Troubleshooting
  section in Administration Guide (bsc#1204195)
```
Jargon is not allowed in `changelog` :)

File: modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
++
+. Log in to {productname} Server.
++
+. Go to [guimenu]``Admin > Setup Wizard > Organization Credentials``.
++
+. Make sure you already have SUSE credentials for {productname}.
++
+. Select option for adding new and type in Microfocus credentials.
++
+. Go to [guimenu]``Admin > Setup Wizard > Organization Credentials``page and wait for the refresh operation to complete.
```
Comment (keichwa) [inline modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc:19]:
```suggestion
. Go to [guimenu]``Admin > Setup Wizard > Organization Credentials`` and wait for the refresh operation to complete.
```

File: modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
++
+. OES should be visible on the refreshed product list. 
+    You can now enable it like any other products.
```
Comment (keichwa) [inline modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc:22]:
```suggestion
  You can now enable it like any other products.
```

File: modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
+[[troubleshooting-oes-repositories-enablement]]
+= Troubleshooting OES repositories enablement
+
+To enable Open Enterprise Server (OES) on {productname} Server, follow the the procedure described.
```
Comment (keichwa) [inline modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc:4]:
```suggestion
To enable Open Enterprise Server (OES) on {productname} Server, follow the procedure described.
```

File: modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+. OES should be visible on the refreshed product list. 
+    You can now enable it like any other products.
+
+
+[IMPORTANT]
+====
+Do not change [path]``additional_products.json`` or [path]``java.product_tree_tag``.
+====
+
+For more details about OES documentation, see https://www.microfocus.com/documentation/open-enterprise-server/.
```
Comment (keichwa) [inline modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc:30]:
```suggestion
For more information about OES, see https://www.microfocus.com/documentation/open-enterprise-server/.
```

**Review by keichwa — CHANGES_REQUESTED** [review]
ok, but next time we can probably change some wording according to usage elsewhere in the documentation.  For example, instead of "go to" we prefer "navigate to" -- Lana introduced it  ;-)

**Review by keichwa — APPROVED** [review]
Thanks, it now LGTM!

File: modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+. Go to [guimenu]``Admin > Setup Wizard > Organization Credentials`` and wait for the refresh operation to complete.
++
+. OES should be visible on the refreshed product list. 
+  You can now enable it like any other products.
+
+
+[IMPORTANT]
+====
+Do not change [path]``additional_products.json`` or [path]``java.product_tree_tag``.
+====
```
Comment (mcalmer) [inline modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc:28]: @0rnela can you remove this? No customer knows about it. It was just our supporters who scanned bugzilla and found comments about it as the OES Development and QA Team need to use it for beta testing when they want to see it before we change it to "released". But let's not confuse the customer with it.

**Pull request merged**